footprints/templates/xeditable/place.html

```diff
@@ -1,12 +1,13 @@
 <script type="text/template" id="xeditable-place-form">
     <div class="editable-place">
         <div class="form-group">
-            <label>Pinpoint the location on a map</label> 
+            <label>Pinpoint the place on a map</label> 
             <div class="help-block alert alert-info">
                 Search by address and select a location, or click the map to drop a marker.
```

Comment:
This mentions dropping a marker, is this correct?

Reply:
Great catch. I will update.